docs/topics/auth/default.txt

```diff
@@ -999,6 +999,9 @@ implementation details see :ref:`using-the-views`.
       authenticated users accessing the login page will be redirected as if
       they had just successfully logged in. Defaults to ``False``.
 
+    * ``success_url_allowed_hosts``: A set of additional hosts that are safe
```

Comment:
I see that `PasswordResetConfirmView.post_reset_login` is an example of how to handle this, so I'll follow that pattern. I added release notes about the attribute, but did not add the `versionadded` annotation as it is below a class with an annotation already.